datalake-api/src/main/java/com/sequenceiq/sdx/api/endpoint/SdxEndpoint.java

```diff
@@ -274,6 +275,12 @@ RangerCloudIdentitySyncStatus getRangerCloudIdentitySyncStatus(@PathParam("envCr
     ObjectStorageValidateResponse validateCloudStorage(@ValidStackNameFormat @ValidStackNameLength @PathParam("name") String clusterName,
             @Valid SdxValidateCloudStorageRequest sdxValidateCloudStorageRequest);
 
+    @POST
+    @Path("validate_cloud_backup_storage/{name}")
+    @Produces(MediaType.APPLICATION_JSON)
+    @ApiOperation(value = "validate cloud backup storage", produces = MediaType.APPLICATION_JSON, nickname = "validateCloudBackupStorage")
+    ValidationResult validateBackupStorage(@ValidStackNameFormat @ValidStackNameLength @PathParam("name") String clusterName);
```

Comment:
Are you sure that you don't want to use SdxValidateCloudStorageRequest as a parameter here, like in 'validateCloudStorage'? This seems very fragile this way.

Reply:
This API might be called by other services. I want to to trigger this API from datalake-dr (thunderhead service) which will not have the information SdxValidateCloudStorageRequest.

Which part of the implementation is fragile?

Reply:
I think that the API design is fragile not the implementation. If you need another input parameter in the future for the validation (if it gets more complex) then you have to change the API, and that is not easy if several services are using this one.

Reply:
I understand your point.

Reply:
addressed.